src/ngraph/op/util/eval_identity.hpp

```diff
@@ -0,0 +1,20 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#define IDENTITY(a)                                                                                \
```

Comment:
IDENTITY makes think of all 1's at first glance. Can we please rename to COPY or IDENTICAL or TYPE_COPY  or something else to convey the copy part?

Reply:
I changed the name to COPY_TENSOR